examples/navier-stokes/densitycurrent.h

```diff
@@ -167,10 +167,10 @@ CEED_QFUNCTION(ICsDC)(void *ctx, CeedInt Q,
 //   Fe = u Fu + k grad( T )
 //
 // Equation of State:
-//   P = (gamma - 1) (E - rho (u u) / 2)
+//   P = (gamma - 1) (E - rho (u u) / 2 - rho g z)
 //
 // Temperature:
-//   T = (E / rho - (u u) / 2) / cv
+//   T = (E / rho - (u u) / 2 - g z) / cv
```

Comment:
I would remove the description of the temperature in terms of energy here, because we already define energy in terms of temperature, so this seems circular.

Reply:
Thank you, @valeriabarra for your review. I agree. I just changed them.